spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/Benchmark.java

```diff
@@ -102,6 +102,7 @@ public void run() throws IOException {
             config.dataScalesGb().forEach(scale -> {
                 log.info("Beginning benchmarks at a new data scale of {}.", SafeArg.of("dataScale", scale));
                 registration.registerTables(scale);
```

Comment:
minor, but can we rename `registerTables` to `registerTpcdsTables` to match the new method naming scheme?